Resources/Catalog/LcgFileCatalogClient.py

```diff
@@ -162,13 +170,11 @@ def returnCode( error, value = '', errMsg = '' ):
 #
 #####################################################
 
-class LcgFileCatalogClient( FileCatalogueBase ):
+class LcgFileCatalogClient( object ):
```

Comment:
Well, as I tried to put in a comment,  FileCatalogClientBase is only for clients that are built in the DIRAC itself, e.g. Transformation or Bookkeeping. Inheriting in the LcgFileCatalogClient I think is misleading with this respect. We do not have a need to implement a predefined interface here. So, I do not see any need for this inheritance.